test/cpp/jit/test_custom_class_registrations.cpp

```diff
@@ -210,6 +246,16 @@ TORCH_LIBRARY(_TorchScriptTesting, m) {
       .def("add", &Foo::add)
       .def("combine", &Foo::combine);
 
+  m.class_<FooGetterSetter>("_FooGetterSetter")
```

Comment:
I think using lambda's as getter and setter already works with your implementation, could we add test cases covering lambdas? Lambdas are really important because TorchScript only supports a subset of C++ types, users frequently rely on lambdas to cast types back-and-forth between TorchScript friendly types and actual C++ types.

Reply:
added a new test.